src/main/java/org/zalando/stups/tokens/FileSupplier.java

```diff
@@ -53,12 +55,15 @@ public static File getCredentialsDir() {
         String dir = System.getenv(CREDENTIALS_DIR_PROP);
         if (dir == null) {
 
-            // this for testing
             dir = System.getProperty(CREDENTIALS_DIR_PROP);
             if (dir == null) {
-                throw new IllegalStateException(
-                        String.format("environment variable %s not set", CREDENTIALS_DIR_PROP)
-                );
+                // to avoid some configuration on developers
+                if (new File(DEFAULT_CREDENTIALS_DIR).exists() && new File(DEFAULT_CREDENTIALS_DIR).isDirectory()) {
+                    return new File(DEFAULT_CREDENTIALS_DIR);
+                } else {
+                    throw new IllegalStateException(
+                            String.format("environment variable %s not set and default '/meta/credentials' not exists or not a directory.", CREDENTIALS_DIR_PROP));
```

Comment:
.. does not exist or is not a directory. Also: put %s and take /meta/credentials from the constant.